src/main/java/org/elasticsearch/indices/store/IndicesStore.java

```diff
@@ -357,7 +416,8 @@ private boolean shardActive(ShardActiveRequest request) {
     }
 
     private static class ShardActiveRequest extends TransportRequest {
-
+        public static final TimeValue DEFAULT_TIMEOUT = new TimeValue(1, TimeUnit.SECONDS);
```

Comment:
I think we want to make this less aggressive, no? how about 30s? also, I wonder if we should make it configurable to be safe and allow us to speed it up in tests (imho it doesn't have to be dynamic) .